CHANGELOG.md

```diff
@@ -16,6 +16,107 @@ Version numbers 0.2.x to 0.7.x were intentionally not used to avoid conflicts wi
+* Added security scheme and scopes for each endpoint / method by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/295
+
+
+### Changed
+* Made + prefix mandatory for phoneNumber by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/301
+* Clarification of concepts and properties related to the management of session duration and session extension by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/296
+* Updated Authorization and Authentication text to ICM release 0.2.0 wording by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/327
+* Aligned quality-on-demand further with Commonalties 0.4.0 by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/326
+  * Added documentation about "Handling of device information" within the info description. 
+  * Made device parameter optional within createSession
+  * Aligned Device object and info object with Commonalities
+  * Updated Schemata for error responses according to Commonalities
+
+### Fixed
+* Removed unresolved documentation reference within quality-on-demand.yaml by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/324
+
+### Removed
+* Removed unused `messages` object in the `SessionInfo` schema by @sfnuser in https://github.com/camaraproject/QualityOnDemand/pull/312
+* Removed TermsOfService and Contact from APIs since they are optional and did not provide useful information by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/311
+* Removed cucumber directory and its content by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/323 
+
+## qos-profiles v0.11.0-rc.1
+
+**qos-profiles v0.11.0-rc.1 is the first release candidate of the version 0.11.0**
+
+qos-profiles 0.11.0 provides the QoS Profiles endpoints from v0.10.1, changed the retrieval operation to allow the get the QoS Profiles available for a given device, and is aligned with Commonalities 0.4.0 and Identity and Consent Management 0.2.0. **There are breaking changes compared to v0.10.1.**
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.1/code/API_definitions/qos-profiles.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.1/code/API_definitions/qos-profiles.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r1.1/code/API_definitions/qos-profiles.yaml)
+
+### Added
+* Added x-correlator header to requests and responses by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/283
+* Added security scheme and scopes for each endpoint / method by @eric-murray in https://github.com/camaraproject/QualityOnDemand/pull/295
+
```

Comment:
We need to add in the breaking change from a get to a post for querying QoS profiles and the ability to query by device.  Would that go under added or changed?

Since it's a breaking change, it would be good to highlight it as such.

Would something like this work?
```
* Added query to qos-profiles to query profiles available on a given device by @RandyLevensalor in https://github.com/camaraproject/QualityOnDemand/pull/318.
NOTE: This update also changes the `/qos-profiles` from a get to a post to support this new capability.
```